From: Takashi Iwai <tiwai@suse.de>
Date: Thu, 5 Jan 2006 17:40:56 +0000 (+0100)
Subject: [ALSA] emu10k1 - Fix the confliction of 'Front' control
X-Git-Url: http://www.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=7eae36fbd5ea9db3d3fe0d671199121be782a5b3

[ALSA] emu10k1 - Fix the confliction of 'Front' control

Modules: EMU10K1/EMU10K2 driver

Fix the confliction of 'Front' controls on models with STAC9758 codec.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---

--- a/sound/pci/emu10k1/emumixer.c
+++ b/sound/pci/emu10k1/emumixer.c
@@ -759,6 +759,8 @@ int __devinit snd_emu10k1_mixer(struct s
 		"Master Mono Playback Volume",
 		"PCM Out Path & Mute",
 		"Mono Output Select",
+		"Front Playback Switch",
+		"Front Playback Volume",
 		"Surround Playback Switch",
 		"Surround Playback Volume",
 		"Center Playback Switch",
